content/quizzes/en.yaml

```diff
@@ -572,7 +572,7 @@ levels:
                 -   "option_text": "You do not need quotation marks when using the {ask} command"
                     "feedback": "Both {print} and {ask} require quotation marks"
                 -   "option_text": "You can choose yourself whether to use quotation marks or not."
-                    "feedback": "Unfortunately, Hedy is stricter than that."
+                    "feedback": "Un{for}tunately, Hedy is stricter than that."
```

Comment:
This too is a misplaced for, maybe do one more manual check of the replacements?